### ISSUE TITLE
Model Improvements

### ISSUE THREAD

**Issue by batson** [czbiohub/noise2self#4]
For nn training, try:

1. [wide activations](https://arxiv.org/pdf/1808.08718.pdf) for resnet blocks
2. [weight norm](https://pytorch.org/docs/stable/_modules/torch/nn/utils/weight_norm.html) instead of batch norm.
